container-chains/templates/frontier/node/src/chain_spec.rs

```diff
@@ -88,23 +88,14 @@ pub fn template_session_keys(
     container_chain_template_frontier_runtime::SessionKeys { aura: keys }
 }
 
-pub fn development_config(para_id: ParaId, seeds: Option<Vec<String>>) -> ChainSpec {
+pub fn development_config(para_id: ParaId) -> ChainSpec {
     // Give your base currency a unit name and decimal places
     let mut properties = sc_chain_spec::Properties::new();
     properties.insert("tokenSymbol".into(), "UNIT".into());
     properties.insert("tokenDecimals".into(), 18.into());
     properties.insert("ss58Format".into(), 42.into());
     properties.insert("isEthereum".into(), true.into());
 
-    let initial_collator_seeds = seeds.unwrap_or(vec!["Alice".to_string(), "Bob".to_string()]);
-    let collator_accounts: Vec<AccountId> = initial_collator_seeds
-        .iter()
-        .map(|seed| get_account_id_from_seed::<ecdsa::Public>(seed))
-        .collect();
-    let collator_keys: Vec<NimbusId> = initial_collator_seeds
-        .iter()
-        .map(|seed| get_collator_keys_from_seed(seed))
-        .collect();
     let mut default_funded_accounts = pre_funded_accounts();
     default_funded_accounts.extend(collator_accounts.clone());
```

Comment:
these will not work with the current implementation. I would say we keep `let initial_collator_seeds= ["Alice".to_string(), "Bob".to_string()]` for now, they will be removed later on, but for now this should work